samples/servicecontrol/adapter-sqlserver/ServiceControlAdapter/SqlServerTransportAdapter/Sales/Database.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Threading.Tasks;
+
+class Database
```

Comment:
Understanding this is supposed to be a database interface, doesn't that look too simplistic? Should we go for a EntityFramework datacontext instead? Even if no, I would rename this class to DataContext and put a comment there that this is a mock database / EF interface.